bins/revme/src/cmd/evmrunner.rs

```diff
@@ -93,9 +98,22 @@ impl Cmd {
             microbench::bench(&bench_options, "Run bytecode", || {
                 let _ = evm.transact().unwrap();
             });
+        } else if self.trace {
+            println!("Traces:");
+            let mut evm = evm
+                .modify()
+                .reset_handler_with_external_context(TracerEip3155::new(
+                    Box::new(std::io::stdout()),
+                ))
+                .append_handler_register(inspector_handle_register)
+                .build();
+
+            let out = evm.transact().map_err(|_| Errors::EVMError)?;
+            println!("Result: {:#?}", out.result);
         } else {
             let out = evm.transact().map_err(|_| Errors::EVMError)?;
             println!("Result: {:#?}", out.result);
+
             if self.state {
```

Comment:
`self.trace` option should be independent of `self.trace`. So it should be put outside of else branch

Reply:
do you mean `independent of self.state`